packages/design-picker/src/components/premium-badge/index.tsx

```diff
@@ -42,20 +44,30 @@ const PremiumBadge: FunctionComponent< Props > = ( {
 
 	const divRef = useRef( null );
 	const [ isPopoverVisible, setIsPopoverVisible ] = useState( false );
+	const [ isHovered, setIsHovered ] = useState( false );
 	return (
 		<div
-			className={ classNames( 'premium-badge', className ) }
+			className={ classNames( 'premium-badge', className, {
+				'premium-badge__compact-animation': shouldCompactWithAnimation,
+				'premium-badge--compact': shouldCompactWithAnimation && ! isHovered,
```

Comment:
I suppose we can simply use `&:hover` here. WDYT?

Reply:
I tried it before but I couldn't make it work by extending the CSS for the version without the animation. I mean without breaking or changing the CSS. Let's iterate on that later.